modules/trusted-firmware-m/Kconfig.peripheral_secure

```diff
@@ -363,6 +363,44 @@ config NRF_GPIOTE0_SECURE
 	  GPIOTE0 is implemented with secure only security.
 	  This option only enables it to be used in the secure image.
 
+config NRF_GPIOTE20_SECURE
+	bool "GPIOTE configured as secure peripheral"
```

Comment:
this and option above should have instances in the name, otherwise they have the same text and is confusing

Reply:
You are right, I updated this